input/pagecontent/2-10-ContentSharing.md

```diff
@@ -42,16 +42,40 @@ Clients wishing to exchange structured information SHALL:
 
-`[FHIR resource]-update` events MAY also update attributes of the anchor context or associated context resources.  For example, a Subscriber might wish to update the `status` attribute of the anchor context resource.
+`[FHIR resource]-update` events MAY also update attributes of the anchor context or associated context resources.  For example, a Subscriber might wish to update the `status` attribute of the anchor context resource.  If any updates are made during the content sharing session to a resource in the `context` array that was present in the `[FHIR resource]-open` event, the updated resources SHALL be present in the `Bundle` resource in the `updates` key.
 
 When a `[FHIR resource]-update` event is received by a Subscriber, the application should respond as is appropriate for its clinical use.  For example, an image reading application may choose to ignore an observation describing a patient's blood pressure.  Since transactional change sets are used during information exchange, no problems are caused by applications deciding to ignore exchanged information not relevant to their function.  However, they should read and retain the `context.versionId` of the anchor context provided in the event for later use.
 
+Resources referenced in a content sharing session SHOULD be accessible.  For example, a `ServiceRequest` referenced in an `ImagingStudy` should either be present in the content or a fullUrl SHOULD be provided.
+
+#### Context Array in Update and Select Events
+
+In addition to the `Bundle` resource in the `updates` key, the context array of `[FHIR resource]-update` and `[FHIR resource]-select` events SHALL include the only the anchor context resource which was present in the context array of the corresponding `[FHIR resource]-open` event.  The `Patient` resource which was present in the context array of the corresponding `[FHIR resource]-open` event MAY be present in `[FHIR resource]-update` events to enable Subscribers to mitigate patient safety concerns.
+
+Resources in the context array of `[FHIR resource]-update` and `[FHIR resource]-select` events SHOULD include only the`resourceType` and `id` of the resource.  Note this constraint usually results in the FHIR resource in `[FHIR resource]-update` and `[FHIR resource]-select` events being an invalid FHIR resource since FHIR resources generally require an attribute in addition to `id` such as a `status` attribute.  However, this constraint ensures that no changes to attributes of the anchor context resource are made through changes to that resource in the context array in `[FHIR resource]-update` and `[FHIR resource]-update` events.
```

Comment:
A reference instead of an invalid for FHIR resource seems wayyy better here.

Reply:
Agreed - removed this section as previous section replaces it.